functions/packages.sh

```diff
@@ -397,7 +398,7 @@ influxdb_grafana_setup() {
   cond_echo ""
   echo "InfluxDB... "
   cond_redirect wget -O - https://repos.influxdata.com/influxdb.key | apt-key add - || FAILED=1
-  echo "deb https://repos.influxdata.com/debian jessie stable" > /etc/apt/sources.list.d/influxdb.list || FAILED=1
+  echo "deb https://repos.influxdata.com/debian $codename stable" > /etc/apt/sources.list.d/influxdb.list || FAILED=1
```

Comment:
Don't have an answer to this but which of these two could we use?
`https://repos.influxdata.com/debian/dists/*` or `https://repos.influxdata.com/ubuntu/dists/*`

Both seams to have all major releases for both Ubuntu and Debian.

Reply:
Maybe just do an `if´ case for the Ubuntu case?